### ISSUE TITLE
Crashing issues fix.

### ISSUE THREAD

**Issue by AfrizalFirmansyah** [Dar9586/NClientV2#765]
Crashing app maybe cant load something.
Just search on fork, maybe someone fix it.

Thanks to him our hentai hero.
https://github.com/shirokun20/NClientV2

**Comment by JR5474921** [comment]
I cant install it. It always returns the error: 
"was not installed propperly because the package conflicts with already existing package."

I would really like to not uninstall and reinstall the app because i have HUNDREDS of downloaded Manga on it. Is there any possibility to fix this without uninstalling my current app before (maybe renaming the App)?

**Comment by Pokilokui** [comment]
There is actually an issue with dpi on your fixed app, I have a big black rectangle at the bottom of my screen
![image](https://github.com/user-attachments/assets/5aacc4e8-daa1-4339-a165-0a7da2528f64)

**Comment by AfrizalFirmansyah** [comment]
> I cant install it. It always returns the error: "was not installed propperly because the package conflicts with already existing package."
> 
> I would really like to not uninstall and reinstall the app because i have HUNDREDS of downloaded Manga on it. Is there any possibility to fix this without uninstalling my current app before (maybe renaming the App)?

You can uninstall and install new version, before that backup (Export data) in setting.

**Comment by AfrizalFirmansyah** [comment]
> There is actually an issue with dpi on your fixed app, I have a big black rectangle at the bottom of my screen ![image](https://private-user-images.githubusercontent.com/101603910/385527872-5aacc4e8-daa1-4339-a165-0a7da2528f64.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE0NzA5MjgsIm5iZiI6MTczMTQ3MDYyOCwicGF0aCI6Ii8xMDE2MDM5MTAvMzg1NTI3ODcyLTVhYWNjNGU4LWRhYTEtNDMzOS1hMTY1LTBhN2RhMjUyOGY2NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExM1QwNDAzNDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMmEzOTM1YzFlZGViYzhjMmVjNWM1YjkwNWUxODA3YzA3YWYyYTFjYzdmZDYxYzc4MDExZGM0ZTkyMDRmOWExJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ajb0uU-vCCmoAGwHLZtt7UQRm5Wqe92HO-MWM0T5fNM)

Go to app info and activate full screen mode.
![Uploading Screenshot_2024-11-13-11-07-01-079_com.miui.securitycenter.jpg…]()

**Comment by JR5474921** [comment]
> > I cant install it. It always returns the error: "was not installed propperly because the package conflicts with already existing package."
> > I would really like to not uninstall and reinstall the app because i have HUNDREDS of downloaded Manga on it. Is there any possibility to fix this without uninstalling my current app before (maybe renaming the App)?
> 
> You can uninstall and install new version, before that backup (Export data) in setting.

How does that work? ... the last tume i re-installed the App, all my downloaded files were gone.

**Comment by aratinjapantube** [comment]
Finally ! I download it and it workable. Hero !

**Comment by Kronos2308** [comment]
> Crashing app maybe cant load something. Just search on fork, maybe someone fix it.
> 
> Thanks to him our hentai hero. https://github.com/shirokun20/NClientV2

#762 I made apull request to the owner no response yet seems very busy 
I also put a debug apk to fix the issues of crashing and .webp not loading

**Comment by AfrizalFirmansyah** [comment]
> > > I cant install it. It always returns the error: "was not installed propperly because the package conflicts with already existing package."
> > > I would really like to not uninstall and reinstall the app because i have HUNDREDS of downloaded Manga on it. Is there any possibility to fix this without uninstalling my current app before (maybe renaming the App)?
> > 
> > 
> > You can uninstall and install new version, before that backup (Export data) in setting.
> 
> How does that work? ... the last tume i re-installed the App, all my downloaded files were gone.

Go offline, open nclient, go to setting scroll down you can see 
Export data
Import data
This feature is to backup and restore data mean your bookmark, status viewer, favorite, etc.

**Comment by Kronos2308** [comment]
Yep the app I upload is the debug version that have a different id 
So the only way is go offline and make a backup of the app data an then restore that data on the debug version 
Also you may have to move the download files since the id is different

**Comment by JR5474921** [comment]
> > > > I cant install it. It always returns the error: "was not installed propperly because the package conflicts with already existing package."
> > > > I would really like to not uninstall and reinstall the app because i have HUNDREDS of downloaded Manga on it. Is there any possibility to fix this without uninstalling my current app before (maybe renaming the App)?
> > > 
> > > 
> > > You can uninstall and install new version, before that backup (Export data) in setting.
> > 
> > 
> > How does that work? ... the last tume i re-installed the App, all my downloaded files were gone.
> 
> Go offline, open nclient, go to setting scroll down you can see Export data Import data This feature is to backup and restore data mean your bookmark, status viewer, favorite, etc.

But doesnt it delete the downloaded files while uninstalling?
(I use the debug version regularily btw. so it should be the same directory already)

**Comment by Liktyan** [comment]
I Installed the new 3.0.6 version nhclient by shirokun20 but when I tried to restore the app with backup I made in 3.0.5 version it restored nothing plz someome reply
to know I am not the one facing this issue

**Comment by AfrizalFirmansyah** [comment]
> I Installed the new 3.0.6 version nhclient by shirokun20 but when I tried to restore the app with backup I made in 3.0.5 version it restored nothing plz someome reply to know I am not the one facing this issue

Maybe something wrong with your backup file, 
When your backup, it take a while until finished, faster if you have good cpu. Just waiting until file size not increase.
Nothing wrong when i restore my data from 3.0.5 to new 3.0.6

**Comment by AfrizalFirmansyah** [comment]
> > > > > I cant install it. It always returns the error: "was not installed propperly because the package conflicts with already existing package."
> > > > > I would really like to not uninstall and reinstall the app because i have HUNDREDS of downloaded Manga on it. Is there any possibility to fix this without uninstalling my current app before (maybe renaming the App)?
> > > > 
> > > > 
> > > > You can uninstall and install new version, before that backup (Export data) in setting.
> > > 
> > > 
> > > How does that work? ... the last tume i re-installed the App, all my downloaded files were gone.
> > 
> > 
> > Go offline, open nclient, go to setting scroll down you can see Export data Import data This feature is to backup and restore data mean your bookmark, status viewer, favorite, etc.
> 
> But doesnt it delete the downloaded files while uninstalling? (I use the debug version regularily btw. so it should be the same directory already)

No, if your setting folder location of downloaded fap material is in /internal/pictures/nhentai
Or
/sdcard/pictures/nhentai
Yes, if your setting folder location of downloaded fap material is in default app data mean /android/data/com.dar.nclientv2/

**Comment by JR5474921** [comment]
> > > > > > I cant install it. It always returns the error: "was not installed propperly because the package conflicts with already existing package."
> > > > > > I would really like to not uninstall and reinstall the app because i have HUNDREDS of downloaded Manga on it. Is there any possibility to fix this without uninstalling my current app before (maybe renaming the App)?
> > > > > 
> > > > > 
> > > > > You can uninstall and install new version, before that backup (Export data) in setting.
> > > > 
> > > > 
> > > > How does that work? ... the last tume i re-installed the App, all my downloaded files were gone.
> > > 
> > > 
> > > Go offline, open nclient, go to setting scroll down you can see Export data Import data This feature is to backup and restore data mean your bookmark, status viewer, favorite, etc.
> > 
> > 
> > But doesnt it delete the downloaded files while uninstalling? (I use the debug version regularily btw. so it should be the same directory already)
> 
> No, if your setting folder location of downloaded fap material is in /internal/pictures/nhentai Or /sdcard/pictures/nhentai Yes, if your setting folder location of downloaded fap material is in default app data mean /android/data/com.dar.nclientv2/

Yeah, my current set download location is the default one you mentioned last. 
Does it transfer my downloaded files automatically to the new location if i update it in settings?
If no: Can i manually transfer the files somehow to make it work smoothly after re-installation? (My device doesnt let me access any data within the folder //.../android/data/     )

**Comment by Norikazen** [comment]
> Crashing the app maybe cant load something. Just search on fork, maybe someone will fix it.
> > Thanks to him our hentai hero. https://github.com/shirokun20/NClientV2

Cannot be installed on Android 15

**Comment by AfrizalFirmansyah** [comment]
> > > > > > > I cant install it. It always returns the error: "was not installed propperly because the package conflicts with already existing package."
> > > > > > > I would really like to not uninstall and reinstall the app because i have HUNDREDS of downloaded Manga on it. Is there any possibility to fix this without uninstalling my current app before (maybe renaming the App)?
> > > > > > 
> > > > > > 
> > > > > > You can uninstall and install new version, before that backup (Export data) in setting.
> > > > > 
> > > > > 
> > > > > How does that work? ... the last tume i re-installed the App, all my downloaded files were gone.
> > > > 
> > > > 
> > > > Go offline, open nclient, go to setting scroll down you can see Export data Import data This feature is to backup and restore data mean your bookmark, status viewer, favorite, etc.
> > > 
> > > 
> > > But doesnt it delete the downloaded files while uninstalling? (I use the debug version regularily btw. so it should be the same directory already)
> > 
> > 
> > No, if your setting folder location of downloaded fap material is in /internal/pictures/nhentai Or /sdcard/pictures/nhentai Yes, if your setting folder location of downloaded fap material is in default app data mean /android/data/com.dar.nclientv2/
> 
> Yeah, my current set download location is the default one you mentioned last. Does it transfer my downloaded files automatically to the new location if i update it in settings? If no: Can i manually transfer the files somehow to make it work smoothly after re-installation? (My device doesnt let me access any data within the folder //.../android/data/ )

Move downloaded files manually.
Install shizuku
Use mixplorer,etc work with shizuku to access android data folder.

**Comment by pokerddd** [comment]
Giving me error when installing

**Comment by shirokun20** [comment]
try with fork code https://github.com/shirokun20/NClientV2/releases/tag/3.0.6

**Comment by mengmeng810416** [comment]
这个3.0.6的app,不能下载本子是为啥，只走流量下不动啊

**Comment by mengmeng810416** [comment]
Why can't I download this 3.0.6 app from my laptop? It only relies on data usage and can't be downloaded

**Comment by mengmeng810416** [comment]
This 3.0.6 app can not download the book, only walk the traffic download is why
